pkgs/development/python-modules/flask-sqlalchemy-lite/default.nix

```diff
@@ -21,6 +22,14 @@ buildPythonPackage rec {
     hash = "sha256-LpdPp5Gp74DSJqD1DJqwNeaMKdN5pEAUkxnKGYZcVis=";
   };
 
+  patches = [
+    # fix python3.13 compat
+    (fetchpatch2 {
```

Comment:
What's the reason for using fetchpatch2? I think it should be avoided if fetchpatch works fine.

Reply:
> What's the reason for using fetchpatch2? I think it should be avoided if fetchpatch works fine.

This:

> In the interest of keeping our maintenance burden and the size of Nixpkgs to a minimum, patches already merged upstream or published elsewhere should be retrieved using fetchpatch2

https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#fetching-patches